modules/odc-importing-a-codebase-from-git-to-create-an-application.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// applications/application-life-cycle-management/odc-creating-an-application-using-the-developer-perspective.adoc
+
+[id="odc-importing-a-codebase-from-git-to-create-an-application_{context}"]
+= Importing a codebase from Git to create an application
+
+The following tutorial walks you through the *Import from Git* option to create an application.
+
+Create, build, and deploy an application on {product-title} using an existing codebase in GitHub as follows:
+
+.Prerequisites
+
+* Ensure that you have link:https://docs.openshift.com/container-platform/latest/web-console/web-console.html[logged in to the web console].
```

Comment:
Edit the link per our guidelines https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#internal-cross-references